src/test/java/plugins/MavenPluginTest.java

```diff
@@ -246,4 +248,20 @@ private void assertHasModule(MavenModuleSet job, String name) {
         find(by.xpath("//a[@href='%s/']", name)).click();
         assertThat(driver.getCurrentUrl(), equalTo(build.module(name).url.toExternalForm()));
     }
+
+    @Issue("JENKINS-22252")
+    @WithPlugins({"maven-plugin@2.12", "tasks"})
```

Comment:
I suppose this version specification is a temporary thing and @jglick is going to update it to 2.13 after the release. Before that this test will be failing.